config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I don't think these changes should be committed (this is the file that is supposed to be git ignored but acts weird)

Reply:
I added a snippet to the readme to prevent tracking this file locally.